les/vflux/server/balance.go

```diff
@@ -256,16 +256,34 @@ func (n *NodeBalance) Priority(now mclock.AbsTime, capacity uint64) int64 {
 // in the current session.
 // If update is true then a priority callback is added that turns UpdateFlag on and off
 // in case the priority goes below the estimated minimum.
-func (n *NodeBalance) EstMinPriority(at mclock.AbsTime, capacity uint64, update bool) int64 {
+func (n *NodeBalance) EstimatePriority(now mclock.AbsTime, capacity uint64, addBalance int64, future, bias time.Duration, update bool) int64 {
 	n.lock.Lock()
 	defer n.lock.Unlock()
 
-	var avgReqCost float64
-	dt := time.Duration(n.lastUpdate - n.initTime)
-	if dt > time.Second {
-		avgReqCost = float64(n.sumReqCost) * 2 / float64(dt)
+	n.updateBalance(now)
```

Comment:
To be honest, we shouldn't pass the `now` here. Instead, we should use the `n.bt.clock.Now()`. It's the same in `Priority`

Reply:
done